ljclang.lua

```diff
@@ -716,6 +749,10 @@ Type_mt.__tostring = Type__index.name
 
 -------------------------------------------------------------------------
 
+function api.clangVersion()
+  return getString(clang.clang_getClangVersion())
```

Comment:
The doc of that function reads
```
Return a version string, suitable for showing to a user, but not
intended to be parsed (the format is not guaranteed to be stable).
```
For example, for me, with 4.0.0 it is:
```
clang version 4.0.0-svn297204-1~exp1 (branches/release_40)
```
What about instead returning the string obtained by `llvm-config --version` when `libljclang_support.so` is built? (It is now there for a cross-check against `libclang`, to be pushed soon.) That one is plainly `MAJOR.MINOR.PATCH`.